src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,96 @@
+.nav {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px $black;
+
+  &__logo {
+    height: 40px;
+  }
+
+  &__link-logo {
+    text-decoration: none;
+    height: 40px;
+  }
+}
+
+.nav__container {
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  text-decoration: none;
+  list-style-type: none;
+
+  .nav__container-list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+  }
+
+  .nav__container-link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding-top: 23px;
+    padding-bottom: 22.5px;
+    color: $black-link;
+
+    &:link {
+      color: $black-link;
+    }
+
+    &:visited {
+      color: $black-link;
+    }
+
+    &:hover {
+      color: $blue;
+    }
+
+    &:focus::after {
+      content: "";
+      display: block;
+      position: absolute;
+      bottom: 0;
+      height: 4px;
+      width: 100%;
+      border-radius: 8px;
+      background-color: $blue;
+    }
```

Comment:
the same as `is-active::after`

Reply:
![image](https://user-images.githubusercontent.com/116836849/205697208-88ecd0d6-2950-4d0d-9e1e-7b9d8c1173c3.png)
not the same, we need focus

Reply:
i not fixed